package.json

```diff
@@ -97,6 +97,12 @@
                     "default": false,
                     "description": "Enable experimental code actions (requires v1.7.0+)."
                 },
+                "fortran-ls.disableDiagnostics": {
+                    "scope": "resource",
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Disable diagnostics (requires v1.11.0+)."
```

Comment:
Not sure if v1.11.0+ will be the next version you'll choose

Reply:
Thanks for the contribution! I just released this feature as part of `v1.12.0`, can you please update this line and then I will merge it in?

Reply:
Done :)